.github/workflows/c3-e2e.yml

```diff
@@ -75,6 +75,24 @@ jobs:
           git config --global user.email wrangler@cloudflare.com
           git config --global user.name 'Wrangler Tester'
 
+      - name: Deploy test worker
```

Comment:
Can we move this to its own `job` and then change the test jobs to use `need: ` to ensure that this job has run first (and only once).